### DISCUSSION TITLE
What does it do ?

### DISCUSSION THREAD

**Discussion by donburch888** [WLAN-Pi/feedback#45] [Feature Request]
Just another neat gadget by engineers for engineers. 

What does it do ?
I has a V8 engine, 16 inch wheels, and a whoosit watchet ! Ain't is great !!!

But what does it DO ?
It doesn't have to do anything, it just looks great, eh ?

**Comment by jiribrejcha** [comment]
Haha :) Have a look here, Don.

http://go.wlanpi.com
